pydis_site/templates/home/index.html

```diff
@@ -98,8 +85,8 @@ <h1 class="is-size-1">Who are we?</h1>
               <i class="fa fa-asterisk"></i>
               <i class="fa fa-code"></i>
               <i class="fab fa-python"></i>
-              <i class="fa fa-alien-monster"></i>
-              <i class="fa fa-duck"></i>
+              <i class="fa fa-alien-monster is-hidden-mobile"></i>
+              <i class="fa fa-duck is-hidden-mobile"></i>
```

Comment:
Similarly, has this been discussed somewhere? I do see the point of this change though.

Reply:
This was also discussed in `#dev-contrib`